ecology.yaml

```diff
@@ -0,0 +1,525 @@
+
+  - name: ggplot2_pca
+    owner: iuc
+    tool_panel_section_label: 'Graph/Display Data'
+
+  - name: pygenometracks
+    owner: iuc
+    tool_panel_section_label: 'Graph/Display Data'
+
+  - name: ggplot2_heatmap
+    owner: iuc
+    tool_panel_section_label: 'Graph/Display Data'
+
+  - name: ggplot2_violin
+    owner: iuc
+    tool_panel_section_label: 'Graph/Display Data'
+
+  - name: ggplot2_heatmap2
+    owner: iuc
+    tool_panel_section_label: 'Graph/Display Data'
+
+  - name: ggplot2_point
+    owner: iuc
+    tool_panel_section_label: 'Graph/Display Data'
+
+  - name: ggplot2_histogram
+    owner: iuc
+    tool_panel_section_label: 'Graph/Display Data'
+
+  - name: tsne
+    owner: iuc
+    tool_panel_section_label: 'Graph/Display Data'
+
+  - name: best_regression_subsets
+    owner: devteam
+    tool_panel_section_label: 'Statistics'
+
+  - name: principal_component_analysis
+    owner: devteam
+    tool_panel_section_label: 'Statistics'
+
+  - name: kernel_principal_component_analysis
+    owner: devteam
+    tool_panel_section_label: 'Statistics'
+
+  - name: kernel_canonical_correlation_analysis
+    owner: devteam
+    tool_panel_section_label: 'Statistics'
+
+  - name: canonical_correlation_analysis
+    owner: devteam
+    tool_panel_section_label: 'Statistics'
+
+  - name: sklearn_svm_classifier
+    owner: bgruening
+    tool_panel_section_label: 'Statistics'
+
+  - name: sklearn_sample_generator
+    owner: bgruening
+    tool_panel_section_label: 'Statistics'
+
+  - name: sklearn_searchcv
+    owner: bgruening
+    tool_panel_section_label: 'Statistics'
+
+  - name: sklearn_build_pipeline
+    owner: bgruening
+    tool_panel_section_label: 'Statistics'
+
+  - name: sklearn_pairwise_metrics
+    owner: bgruening
+    tool_panel_section_label: 'Statistics'
+
+  - name: sklearn_nn_classifier
+    owner: bgruening
+    tool_panel_section_label: 'Statistics'
+
+  - name: sklearn_numeric_clustering
+    owner: bgruening
+    tool_panel_section_label: 'Statistics'
+
+  - name: sklearn_ensemble
+    owner: bgruening
+    tool_panel_section_label: 'Statistics'
+
+  - name: sklearn_feature_selection
+    owner: bgruening
+    tool_panel_section_label: 'Statistics'
+
+  - name: sklearn_data_preprocess
+    owner: bgruening
+    tool_panel_section_label: 'Statistics'
+
+  - name: sklearn_clf_metrics
+    owner: bgruening
+    tool_panel_section_label: 'Statistics'
+
+  - name: sklearn_regression_metrics
+    owner: bgruening
+    tool_panel_section_label: 'Statistics'
+
+  - name: sklearn_discriminant_classifier
+    owner: bgruening
+    tool_panel_section_label: 'Statistics'
+
+  - name: sklearn_generalized_linear
+    owner: bgruening
+    tool_panel_section_label: 'Statistics'
+
+  - name: sklearn_model_validation
+    owner: bgruening
+    tool_panel_section_label: 'Statistics'
+
+  - name: scipy_sparse
+    owner: bgruening
+    tool_panel_section_label: 'Statistics'
+
+  - name: sklearn_suite
```

Comment:
Can you please list all tools of this suite and not the suite alone.
Please note that adding those here is not strictly needed as we do have those tools already installed.

Reply:
Done! (But I think that you made it no ? ;) )